chaosext/actions.py

```diff
@@ -0,0 +1,8 @@
+
+# -*- coding: utf-8 -*-
+import json
+import os.path
+from typing import Union
+
+from chaoslib.exceptions import FailedActivity
+from chaoslib.types import MicroservicesStatus, Secrets
```

Comment:
drop the `MicroservicesStatus`. It'll be gone soon.

However, add `Configuration` instead please:

```
from chaoslib.types import Configuration, Secrets
```

Reply:
Will do